dapr/proto/daprclient/v1/daprclient.proto

```diff
@@ -53,7 +53,12 @@ message BindingResponseEnvelope {
 
 
 message GetTopicSubscriptionsEnvelope {
-  repeated string topics = 1;
+  repeated TopicSubscriptionEnvelope subscriptions = 1;
+}
+
+message TopicSubscriptionEnvelope {
+  string topic = 1;
```

Comment:
where is this var "topic" used?

Reply:
Its the proto file that is generated into the gRPC Go client.

Used here:
https://github.com/dapr/dapr/pull/1529/files#diff-a6a78e03551b53fec882ef23c9762278R760